recipe/meta.yaml

```diff
@@ -1,16 +1,24 @@
 {% set name = "numcodecs" %}
-{% set version = "0.12.1" %}
+{% set version = "0.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 05d91a433733e7eef268d7e80ec226a0232da244289614a8f3826901aec1098e
+  sha256: ba4fac7036ea5a078c7afe1d4dffeb9685080d42f19c9c16b12dad866703aa2e
+  patches:
+    ################################################################
+    # Revert NumPy build dependency                                #
+    #                                                              #
+    # xref: https://github.com/zarr-developers/numcodecs/pull/557  #
+    ################################################################
+    - patches/PR_557.patch
```

Comment:
Thanks Ryan! 🙏

Have used the reversion patch PR: https://github.com/zarr-developers/numcodecs/pull/557

Let's see if this helps